ci-operator/step-registry/aks/provision/aks-provision-commands.sh

```diff
@@ -27,6 +49,8 @@ AKE_CREATE_COMMAND=(
     --node-count "$AKS_NODE_COUNT"
     --load-balancer-sku "$AKS_LB_SKU"
     --os-sku "$AKS_OS_SKU"
+    "${CLUSTER_AUTOSCALER_ARGS:-}" \
+    "${K8S_VERSION_ARGS:-}" \
```

Comment:
Would the command fail when these args are null strings ?

Reply:
No should proceed successfully, I will run one of your rehearsal jobs just to double check.